### ISSUE TITLE
Handle non-default SQLAlchemy binds

### ISSUE THREAD

**Issue by mcsoini** [Flask-Middleware/flask-security#738]
TL;DR: In order to store users in a non-default database, I'm monkey patching the ``fsqla.FsModels`` class to set the ``bind_key`` parameter of the ``roles_users`` table. Is there a better way or would this make sense as a PR? 

--------------------------------------
I am storing users in a non-default sqlalchemy database `usersdb`, defined in SQLALCHEMY_BINDS. I am setting up the User and Role tables as follows:
``` python
class Role(db.Model, fsqla.FsRoleMixin):
    __bind_key__ = "usersdb"

class User(db.Model, fsqla.FsUserMixin):
    __bind_key__ = "usersdb"

user_datastore = SQLAlchemyUserDatastore(db, User, Role)
````

However, this would give me a ``sqlalchemy.exc.NoReferencedTableError``, since the ``roles_users`` table refers to the default database. I therefore proceeded to monkey patch the ``fsqla.FsModels`` class as follows:

``` python
class FsModelsBind(fsqla.FsModels):
    """ Class to monkey patch the ``fsqla.FsModels`` class, adding a 
    ``bind_key`` parameter to the ``roles_users`` table.
    """

    @classmethod
    def set_db_info(
        cls,
        appdb,
        user_table_name="user",
        role_table_name="role",
        webauthn_table_name="webauthn",
        bind_key=None  # <= this!
    ):
        """Initialize Model.
        This needs to be called after the DB object has been created
        (e.g. db = Sqlalchemy())
        """
        cls.db = appdb
        cls.user_table_name = user_table_name
        cls.role_table_name = role_table_name
        cls.webauthn_table_name = webauthn_table_name
        cls.roles_users = appdb.Table(
            "roles_users",
            db.Column("user_id", db.Integer(), db.ForeignKey(f"{cls.user_table_name}.id")),
            db.Column("role_id", db.Integer(), db.ForeignKey(f"{cls.role_table_name}.id")),
            bind_key=bind_key  # <= this!
        )


fsqla.FsModels = FsModelsBind
fsqla.FsModels.set_db_info(db, bind_key="usersdb")
```

This works beautifully, but remains a monkey patch. Is there a better way?

**Comment by jwag956** [comment]
This looks harmless and doesn't affect any current users - so I'd be fine with a PR

**Comment by jwag956** [comment]
Open to PR  - otherwise seems like a very app specific concern.
